app/include/zmk/events/led_indicator_changed.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2021 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#pragma once
+
+#include <zmk/led_indicators.h>
```

Comment:
Since we only need this for the flags type, not any implementation of the functions, can we move the flag type alias to something like:

```suggestion
#include <zmk/led_indicators_types.h>
```